src/aui/tabart.cpp

```diff
@@ -284,6 +285,11 @@ void wxAuiGenericTabArt::DrawBackground(wxDC& dc,
     int topLightness = 90;
     int bottomLightness = 170;
 
+    if (wxSystemSettings::GetAppearance().IsUsingDarkBackground())
+    {
+        wxSwap(bottomLightness, topLightness);
```

Comment:
This needs to set the topLightness to 110, not 170.   170 is way too light for dark mode.  It's a very light gray.

Reply:
Oh, I overlooked this, thanks for noticing, will fix.